tests/QMLEngine/imports.js

```diff
@@ -28,5 +28,17 @@ describe('QMLEngine.imports', function() {
     var qml = load("NoQmldir", this.div);
     expect(qml.value).toBe(67);
   });
+  it("imports are local to file, should succeed", function() {
+    var f = function() {
+      load("LocalToFile/Succeed", this.div);
+    };
+    expect(f.bind(this)).not.toThrowError("No constructor found for WebSocket");
```

Comment:
Does this need to be so specific? Perhaps just expect it not to throw any error?